lib/plugins.ts

```diff
@@ -1,7 +1,15 @@
-var webdriver = require('selenium-webdriver');
-import {Logger} from './logger';
 import * as Q from 'q';
+import * as webdriver from 'selenium-webdriver';
+
 import {ConfigParser} from './configParser';
+import {Logger} from './logger';
+
+declare module 'q' {
```

Comment:
I did submit a [PR](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/12358) to DefinitelyTyped.